README.md

```diff
@@ -466,3 +496,371 @@ If you want to re-run the pipeline again, you can use the following short-hand c
+
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: TriggerBinding
+metadata:
+  name: vote-app-pipelinebinding
+spec:
+  params:
+  - name: git-revision
+    value: $(body.head_commit.id)
+  - name: namespace
+    value: pipelines-tutorial
+  - name: git-repo-url
+    value: $(body.repository.url)
+  - name: git-repo-name
+    value: $(body.repository.name)
+```
+The exact paths (keys) of parameter we need can be found by examining the event payload (eg: GitHub events).
+
+
+Run following command to apply Triggertemplate.
+
+```bash
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/01_binding.yaml
+```
+>**Note**:- The parameter namespace is hardcoded as `pipelines-tutorial`. It can be also parameterized.
+
+#### Event Listener
+
+This component sets up a Service and listens for events. It also connects a TriggerTemplate to a TriggerBinding, into an
+  [addressable](https://github.com/knative/eventing/blob/master/docs/spec/interfaces.md)
+  endpoint (the event sink) 
+
+The definition for our EventListener can be found in
+`03-triggers/03_event_listener.yaml`.
+
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: EventListener
+metadata:
+  name: voteapp-listener
+spec:
+  serviceAccountName: pipeline
+  triggers:
+    - name: vote-app-trig
+      bindings:
+      - name: vote-app-pipelinebinding
+      template:
+        name: vote-app-triggertemplate
+```
+* Run following command to apply Triggertemplate.
+
+```bash
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/03_event_listener.yaml
+```
+
+>***Note***: EventListener will setup a Service. We need to expose that Service as an OpenShift Route to make it publicly accessible.
+
+* Run below command to expose eventlistener service as a route
+
+```bash
+OC expose svc el-voteapp-listener
+```
+
+
+
+
+## Configuring GitHub WebHooks
+
+Now We need to configure webhook-urls on [backend](https://github.com/openshift-pipelines/vote-api) and [frontend](https://github.com/openshift-pipelines/vote-ui) source code repositories with the Route we exposed in the previously.
+
+* Run below command to get webhook-url
+
+```bash
+echo "URL: $(oc  get route el-voteapp-listener --template='http://{{.spec.host}}')"
+```
+
+>***Note:*** 
+>
+>Fork the [backend](https://github.com/openshift-pipelines/vote-api) and [frontend](https://github.com/openshift->pipelines/vote-ui) source code repositories so that you have sufficient privileges to configure GitHub webhooks.
+
+
+### Configure webhook manually
+
+Open forked github repo (Go to Settings > Webhook) 
+click on `Add Webhook` > Add 
+```bash
+echo "$(oc  get route el-voteapp-listener --template='http://{{.spec.host}}')"
+```
+
+to payload URL > Add secret eg: `1234567` > Add Webhook 
+
+![Add webhook](docs/images/add-webhook.png)
+
+- Follow above procedure to configure webhook on [frontend](https://github.com/openshift-pipelines/vote-ui) repo
+
+OR
```

Comment:
I suggest to only leave manual instruction in README and move automatic way into a separate document (and link it from here).